client_plugin/utils/refcount/refcnt.go

```diff
@@ -387,7 +380,7 @@ func (r *RefCountsMap) discoverAndSync(c *client.Client, d drivers.VolumeDriver)
 		log.Debugf("  Mounts for %v", ct.Names)
 		for _, mount := range containerJSONInfo.Mounts {
 			// check if the mount location belongs to vmdk plugin
-			if isVMDKMount(mount.Source) != true {
+			if isVMDKMount(mount.Driver) != true {
```

Comment:
In which case lets leave this as is and I'm going to close this as a won't fix. If user wishes to alias the plugin as _pivotal_ thats fine. As mentioned in the issue its not going to be common practice till a user does do it given Docker allows the daemon to run with a _-g_ option to locate the daemon's files.